src/angular/components/register.component.ts

```diff
@@ -137,6 +137,11 @@ export class RegisterComponent {
             this.platformUtilsService.showToast('success', null, this.i18nService.t('newAccountCreated'));
             this.router.navigate([this.successRoute], { queryParams: { email: this.email } });
         } catch { }
+
+        if (this.hint === this.masterPassword) {
+            this.platformUtilsService.showToast('error', 'Invalid password hint', 'Your password hint cannot be the same as your password.');
+            return;
+        }
```

Comment:
this needs to be up higher, where all the other validation checks are.

Reply:
Updated

Reply:
OK, looks good. Last thing now is these strings need to be localized. They can't be hard coded like this. See the others above for an example. We then need to add the localized keys to each project (browser, web, desktop) under the English file so that they can be translated.

Reply:
Updated, adding links for the localized strings

- https://github.com/bitwarden/web/pull/701

Reply:
I don't see you referencing those strings here?

Reply:
Whoops, I must have pushed the wrong version with git. I'll fix it after work

Reply:
Should be correct now. I'm not sure what went wrong with git but the changes I meant to push were just sitting locally so maybe I just forgot.